kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/src/main/java/org/jbpm/springboot/services/SpringKModuleDeploymentService.java

```diff
@@ -50,7 +53,16 @@ protected RegisterableItemsFactory getRegisterableItemsFactory(AuditEventBuilder
         return factory;
     }
 
-    
+    protected Map<String, Object> buildContextParameters(KieContainer kieContainer) {
```

Comment:
Though it's not mandatory, perhaps it's more clear if "Override" annotation is explicitly shown.

Reply:
done